# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../gimp-hdrtools/makefile.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- gimp-hdrtools-0.1/Makefile_orig	2007-06-15 18:07:12.000000000 +0100
+++ gimp-hdrtools-0.1/Makefile	2007-06-15 18:08:21.000000000 +0100
@@ -24,7 +24,7 @@
 	rm -f $(OBJS) $(TARGET)
 	
 install: all
-	$(GIMPTOOL) --install-bin $(TARGET)
+	$(GIMPTOOL) --install-admin-bin $(TARGET)
 		
 .c.o:
 	$(CC) -c $(CFLAGS) $<
